### ISSUE TITLE
The training configuration of the Faster RCNN on PASCAL VOC

### ISSUE THREAD

**Issue by Alexrich961210** [ZiweiWangTHU/BiDet#30]
Thanks for providing your source code.

My problem is that I use the default training configuration in the Faster-RCNN/trainval_net.py, which trains model 50 epochs and decay the learning rate every 6 epochs. However, I only achieved 17.18% mAP on test set.

How many epochs do you choose for training? Thanks for your time. I am waiting for your reply.

**Comment by Wuziyi616** [comment]
Try decay the lr less frequent, like every 12 epochs? The training of BNNs is very unstable I should say.
